src/issue-strategies/index.js

```diff
@@ -1,17 +1,15 @@
 import * as bugMtStrat from './bug-maintenance.js';
-import * as initStrat from './initiative.js';
-import * as epicStrat from './epic.js';
 import * as storyStrat from './story.js';
-import * as taskStrat from './task.js';
+import * as notAllowedStrat from './not-allowed.js';
 
 export default {
-  Initiative: initStrat,
-  Epic: epicStrat,
+  Initiative: notAllowedStrat,
+  Epic: notAllowedStrat,
```

Comment:
Rather than have the not allowed strategy be duplicated in entries here several times, and this tool requiring a change for each new issue type that gets created, (if any do get created), or trying to create an exhaustive list of every ticket type in Jira, I'd rather see on the `promise.all(promiseList.map())` that happens in the issue-handler some kind of special case or catch that will automatically apply the not allowed strategy on any ticket that isn't found on this object.